core/src/components/table/table-body-row/table-body-row.tsx

```diff
@@ -115,24 +92,21 @@ export class TdsTableBodyRow {
     });
   }
 
-  componentDidLoad() {
-    this.internalTdsPagination.emit(this.tableId);
-  }
-
   render() {
     return (
       <Host
         class={{
           'tds-table__row': true,
+          'tds-table__row--selected': this.selected,
```

Comment:
Not ideal that we're styling the Host elements. These classes are basically part of our public API now, and much of the styling are not encapsulated inside shadow DOM.

We might want to warn developers somehow that undocumented classes like `tds-table__row` might be removed without notice.

Reply:
Agree that it's not ideal. Would require some work to remove it though.. :/